nncf/quantization/algorithms/weight_compression/weight_lowering.py

```diff
@@ -295,7 +300,11 @@ def calculate_quantized_weight(
     level_low = 0 if asym_quant else -(2 ** (num_bits - 1))
     level_high = 2**num_bits - 1 if asym_quant else 2 ** (num_bits - 1) - 1
 
-    compressed_weights = weight / scale
+    if invert_scale:
```

Comment:
What's your plan? I suggest checking the possibility of making `invert_scale=True` by default.

Reply:
Yes, but in the next PR.

Reply:
Please, submit a ticket for this task.